installing/installing_alibaba/installing-alibaba-assisted-installer.adoc

```diff
@@ -0,0 +1,24 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-alibaba-assisted-installer"]
+= Installing on Alibaba Cloud using the Assisted Installer
+include::_attributes/common-attributes.adoc[]
+:context: install-cluster-alibaba
+
+toc::[]
+
+From {product-title} 4.16 and later, you can use the {ai-full} to install an {product-title} cluster on Alibaba Cloud. Alibaba Cloud provides a broad range of cloud computing and data storage services to online businesses and  global enterprises.
+
+:FeatureName: {alibaba} installation with {ai-full}
+include::snippets/technology-preview.adoc[]
+
+include::modules/alibaba-ai-installing.adoc[leveloffset=+1]
+
+.Additional resources
+
+* For the Assisted Installer procedures, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html/installing_openshift_container_platform_with_the_assisted_installer/index[Installing {product-title} with the {ai-full}]
+
+* For the Alibaba Cloud procedures, see link:https://www.alibabacloud.com/blog/alibaba-cloud-red-hat-openshift-container-platform-4-6-deployment-guide_597599[Alibaba Cloud Red Hat {product-title} 4.6 Deployment Guide]
+
+* For instructions on converting the Discovery image to `QCOW2` format, see xref:alibaba-ai-converting-image-to-qcow2_installing-alibaba-assisted-installer[Converting the Discovery image to QCOW2 format].
```

Comment:
I strongly recommend that you remove this link. The [modular docs reference guide](https://redhat-documentation.github.io/modular-docs/#assembly-guidelines) has helpful info about what to include in "Additional resources":

> Focus on relevant resources that might interest the user. Do not list resources for completeness. 

I do not think it is desirable to include an xref to a module that appears in the same assembly. Since there are only 2 modules in this assembly, the user will have no difficulty finding the procedure.

Reply:
The alternative is to say 'below', and that's not an option either:

https://redhat-documentation.github.io/supplementary-style-guide/#cross-references

![Pasted image](https://github.com/openshift/openshift-docs/assets/103109052/73588d36-2f24-478f-866f-7c45420d6e44)